kornia/augmentation/augmentation.py

```diff
@@ -203,17 +201,17 @@ class ColorJitter(AugmentationBase):
     """
 
     def __init__(
-        self, brightness: Union[torch.Tensor, float, Tuple[float, float], List[float]] = 0.,
```

Comment:
Wouldn't drop torch tensor here, otherwise we'll differentiability feature here. One might want to learn or tune some parameters

Reply:
In that case, I think we shall allow tensor params for all augmentation functions. Am I right?

Reply:
Absolutely